tests/expected_specfiles/python-numpy.spec

```diff
@@ -9,7 +9,7 @@ License:        BSD-3-Clause
 URL:            https://www.numpy.org
 Source:         %{pypi_source numpy}
 
-BuildRequires:  python3-devel
+BuildRequires:  python%{python3_pkgversion}-devel
```

Comment:
I am not particularly happy to see `%{python3_pkgversion}` in Fedora spec files. It is not *forbidden* but ti is a spec clutter.

Reply:
Mitigated in 0218b28c1b6eb6dc5192975022bf1857f9e8534c